doc/ipython-notebooks/classification/Classification.ipynb

```diff
@@ -693,7 +693,7 @@
    ],
    "source": [
     "# create Gaussian kernel with width = 2.0\n",
-    "kernel = sg.create_kernel(\"GaussianKernel\", log_width=np.log(2))\n",
+    "kernel = sg.create_kernel(\"GaussianKernel\", width=2)\n",
```

Comment:
but are these the same underlying values?

Reply:
ah no, because there is the weird transformation..

Reply:
I think in most cases it is fine .... just make sure to have an eye on the plots/results in the notebooks, and on the accuracy/MSE in meta examples